File: 1318 Minimum Flips to Make a OR b Equal to c/Minimum Flips to Make a OR b Equal to c.js
function minFlips(a, b, c) {
  let counter = 0;

  while (a > 0 || b > 0 || c > 0) {
    let binA = a & 1;
    let binB = b & 1;
    let binC = c & 1;

    if (binC == 0) {
      counter += binA + binB;
    } else if (binA == 0 && binB == 0) {
      counter += 1;
    }

    a >>= 1;
    b >>= 1;
    c >>= 1;
  }

  return counter;
}

File: 1318 Minimum Flips to Make a OR b Equal to c/Minimum Flips to Make a OR b Equal to c.py
class Solution:
    def minFlips(self, a: int, b: int, c: int) -> int:
        counter =0
        

        while a > 0 or b>0 or c>0:
            binA = a &1
            binB = b &1
            binC = c&1

            if binC == 0:
                counter += (binA + binB)
            elif binA == 0 and binB == 0:
                counter +=1
            
            a >>=1
            b >>= 1
            c >>=1
            
        return counter



